alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -454,7 +454,7 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
             publishFinding(monitor, monitorCtx, finding)
         } catch (e: Exception) {
             // suppress exception
-            logger.error("Optional finding callback failed", e)
+            logger.info("Optional finding callback failed: ${e.message}")
```

Comment:
Should this be changed to debug? This would occur every time a doc level monitor is running even if the user never using security analytics plugin as its currently the only plugin to use this.